backend/routes/accountRoutes.js

```diff
@@ -103,6 +57,98 @@ accountRouter.put("/password", verifyCsrf, passport.authenticate("jwt-strategy",
     }
 });
 
+/**
+ * @route   POST /api/account/delete
+ * @desc    Delete the current user's account
+ * @access  Private
+ * @param   {string} password - The user's password
+ * @returns {object} message - A message indicating whether the account was deleted successfully
+ */
+accountRouter.post('/delete', verifyCsrf, passport.authenticate("jwt-strategy", { session: false }), async (req, res, next) => {
```

Comment:
technically this shouldn't be a post, it should be a delete, and the URI can still be the /:id

Reply:
The only reason I kept it POST was to allow the password to be passed in the request body so that we could verify the password was correct before deleting, backing the "confirmation" functionality to make sure the user/someone else couldn't mistakenly delete the account